Fedora 33: test failed (v1.0.0)

```
+ cd journald-1.0.0
+ LDFLAGS=' -X github.com/ssgreg/journald/version=1.0.0'
+ GO_TEST_FLAGS='-buildmode pie -compiler gc'
+ GO_TEST_EXT_LD_FLAGS='-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld  '
+ go-rpm-integration check -i github.com/ssgreg/journald -b /home/safarov/rpmbuild/BUILD/journald-1.0.0/_build/bin -s /home/safarov/rpmbuild/BUILD/journald-1.0.0/_build -V 1.0.0-1.fc33 -p /home/safarov/rpmbuild/BUILDROOT/golang-github-ssgreg-journald-1.0.0-1.fc33.x86_64 -g /usr/share/gocode -r '.*example.*'
Testing    in: /home/safarov/rpmbuild/BUILD/journald-1.0.0/_build/src
         PATH: /home/safarov/rpmbuild/BUILD/journald-1.0.0/_build/bin:/home/safarov/.local/bin:/home/safarov/bin:/home/safarov/.local/bin:/home/safarov/bin:/home/safarov/.local/bin:/home/safarov/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin
       GOPATH: /home/safarov/rpmbuild/BUILD/journald-1.0.0/_build:/usr/share/gocode:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags " -X github.com/ssgreg/journald/version=1.0.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld  '"
      testing: github.com/ssgreg/journald
github.com/ssgreg/journald


--- FAIL: TestCheckSmallMessage (5.13s)
    send_test.go:79: 
        	Error Trace:	send_test.go:79
        	Error:      	Received unexpected error:
        	            	exit status 1
        	Test:       	TestCheckSmallMessage
```

--- a/send_test.go
+++ b/send_test.go
@@ -75,7 +75,7 @@ func TestCheckSmallMessage(t *testing.T) {
 	require.True(t, strings.Contains(string(out), "MESSAGE=SmallMessage"))
 	require.True(t, strings.Contains(string(out), "PRIORITY=4"))
 
-	out, err = exec.Command("sh", "-c", fmt.Sprintf("journalctl -o verbose 'TEST_ID=%s' -F WITH_NEWLINES", uniqueSmallMessageID)).Output()
+	out, err = exec.Command("sh", "-c", fmt.Sprintf("journalctl -o verbose -t 'TEST_ID=%s' -F WITH_NEWLINES", uniqueSmallMessageID)).Output()
 	require.NoError(t, err)
 	require.Equal(t, "b\n\nsd\n"+"\n", string(out))
 }

